generators/chaincode/templates/javascript/lib/contract-a.js

```diff
@@ -15,20 +15,31 @@
  * limitations under the License.
  */
 
-import { ChaincodeInterface, ChaincodeStub, Shim } from 'fabric-shim';
+'use strict';
 
-export class Chaincode implements ChaincodeInterface {
+const { Contract } = require('fabric-contract-api');
 
-    public async Init(stub: ChaincodeStub): Promise<any> {
-        const { fcn, params } = stub.getFunctionAndParameters();
-        console.info('Init()', fcn, params);
-        return Shim.success();
+const assetType = 'WIDGETS';
+
+class MyContract extends Contract {
+
+    constructor(){
+        super('org.example.mycontract');
```

Comment:
Why is the MyContract contract in the mycontract namespace? Couldn't it just be `org.example`?